zeroex/orderwatch/order_watcher.go

```diff
@@ -702,8 +757,7 @@ func (w *Watcher) generateOrderEventsIfChanged(ordersColTxn *db.Transaction, ord
 			}
 			orderEvents = append(orderEvents, orderEvent)
 		} else if oldFillableAmount.Cmp(newFillableAmount) == 0 {
-			// No important state-change happened, simply update lastUpdated timestamp in DB
-			w.updateOrderDBEntry(ordersColTxn, order)
+			// No important state-change happened
 		} else if oldFillableAmount.Cmp(big.NewInt(0)) == 1 && oldAmountIsMoreThenNewAmount {
```

Comment:
We don't need to necessarily address it right now, but we should go over all of our usage of the `math/big` package to see if there is room for optimization. For example, every occurrence of `big.NewInt(0)` allocates a non-trivial amount of memory and we could reduce the number of allocations by using a single variable instead.

Reply:
Agreed. Can you open an issue for it?

Reply:
Done: https://github.com/0xProject/0x-mesh/issues/473